kubernetes/resource_kubernetes_persistent_volume_v1.go

```diff
@@ -240,14 +240,22 @@ func resourceKubernetesPersistentVolumeV1Create(ctx context.Context, d *schema.R
 			}
 
 			statusPhase := fmt.Sprintf("%v", out.Status.Phase)
-			log.Printf("[DEBUG] Persistent volume %s status received: %#v", out.Name, statusPhase)
+			statusMessage := fmt.Sprintf("%v", out.Status.Message)
+			if statusMessage == "" {
+				log.Printf("[DEBUG] Persistent volume %s status received: %#v", out.Name, statusPhase)
+			} else {
+				log.Printf("[DEBUG] Persistent volume %s status received: %#v, message received: %#v", out.Name, statusPhase, statusMessage)
+			}
 			return out, statusPhase, nil
 		},
 	}
 	_, err = stateConf.WaitForStateContext(ctx)
 	if err != nil {
 		return diag.FromErr(err)
 	}
+	if out.Status.LastPhaseTransitionTime != nil {
```

Comment:
this was added for create, read and delete

Reply:
Looking over this it seems like the use of `out.Status.LastPhaseTransitionTime` would make sense to have in `retry.StateChangeConf` since that's where the Phase is being updated. What was the reason to include this outside of retry?

Reply:
I put it here because I thought it will report the transition time between pending and available/bound. 
During my test I have difficulty of trick pv into pending status (so the retry never happened), so my test is not comprehensive. I will dig a bit more and get back with an updated PR!

Reply:
Having a cluster with only 1 node makes it difficult to have that `Pending` status show up. I was able to get it to show by setting up a 3 node cluster using minikube by running `minikube start --nodes 3 multinode-cluster`

Reply:
Hi @BBBmau ,

I've been caught up with personal commitments lately. Appreciate your recommendation on minikube; it worked wonders! Here are the logs I managed to generate:

```
2024-03-07T18:37:59.947+1100 [DEBUG] provider.terraform-provider-kubernetes: 2024/03/07 18:37:59 [DEBUG] Persistent volume example-pv status received: "Pending"
2024-03-07T18:37:59.947+1100 [DEBUG] provider.terraform-provider-kubernetes: 2024/03/07 18:37:59 [DEBUG] Persistent volume last phrase transition time: 2024-03-07 18:37:59 +1100 AEDT
... 
2024-03-07T18:57:04.439+1100 [DEBUG] provider.terraform-provider-kubernetes: 2024/03/07 18:57:04 [DEBUG] Persistent volume example-pv status received: "Available"
2024-03-07T18:57:04.439+1100 [DEBUG] provider.terraform-provider-kubernetes: 2024/03/07 18:57:04 [DEBUG] Persistent volume last phrase transition time: 2024-03-07 18:57:04 +1100 AEDT
```

I foresee a potential issue where if the refresh code block triggers the sequence PENDING->PENDING->PENDING->AVAILABLE, it might record the same last phrase transition time thrice during the PENDING state. Could this pose a problem? Otherwise, I hope the PR is working as expected 😃